docs/code-quality/ca1501.md

```diff
@@ -49,3 +49,20 @@ The following example shows a type that violates the rule:
 
 [!code-csharp[FxCop.Maintainability.ExcessiveInheritance#1](../code-quality/codesnippet/CSharp/ca1501-avoid-excessive-inheritance_1.cs)]
 [!code-vb[FxCop.Maintainability.ExcessiveInheritance#1](../code-quality/codesnippet/VisualBasic/ca1501-avoid-excessive-inheritance_1.vb)]
+
+## Configurability
```

Comment:
Please add a small sentence about configurability at the end of the `Cause` section above in the document, with a hyperlink to this section. Similar to the last sentence in https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1802?view=vs-2019#cause

Reply:
Also all the existing pages have "Configurability" section above the "Example" section. Please retain this order.